src/index.html

```diff
@@ -10,8 +10,8 @@
   <link rel="icon" type="image/x-icon" href="assets/icon/favicon.ico">
   <link rel="manifest" href="manifest.json">
   <meta name="theme-color" content="#4e8ef7">
-
   <script src="assets/epubjs/build/epub.min.js"></script>
+  <script src="assets/epubjs/build/libs/zip.min.js"></script>
```

Comment:
yes, you are right, I added this line if somebody wants to read an epub file instead of extracted folder.